new/metadata-gem-installation.md

```diff
@@ -0,0 +1,43 @@
+---
+RFC: unassigned
+Author: Lamont Granquist <lamont@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Title
+
+Support a 'gem' DSL method for cookbook metadata to create a dependency on a rubygem.  The
+gem will be installed via `chef_gem` after all the cookbooks are synchronized but before any
+other cookbook loading is done.
+
+## Motivation
+
+    As a Chef User,
+    I want to be able to use additional gems in libraries, attributes and resources,
+    So I don't pull my hair/beard out in frustration.
```

Comment:
As a Chef User,
I want to be able to use additional gems in libraries, attributes and resources,
to avoid complex workarounds and double-run converges.